infra/base-images/base-runner/test_all

```diff
@@ -24,8 +24,17 @@ for FUZZER_BINARY in $(find $OUT/ -executable -type f); do
   fi
 
   FUZZER=$(basename $FUZZER_BINARY)
+  if echo "$FUZZER" | grep "^afl-" > /dev/null 2>&1; then
+    continue
+  fi
+
   echo "testing $FUZZER"
-  run_fuzzer $FUZZER -max_total_time=10
+  if [[ "$FUZZING_ENGINE" = libfuzzer ]]; then
+    run_fuzzer $FUZZER -max_total_time=10
+  else
+    export AFL_NO_UI=1
```

Comment:
Why export, can't we just use at start of the next command OR just embed this inside run_fuzzer ?